src/tree/registries/gitLab/GitLabRepositoryTreeItem.ts

```diff
@@ -17,6 +17,10 @@ export class GitLabRepositoryTreeItem extends RemoteRepositoryTreeItemBase {
     private _nextLink?: string;
 
     public constructor(parent: GitLabProjectTreeItem, id: string, name: string) {
+        if (!name) {
```

Comment:
Can you add a comment as to why? e.g.
`// GitLab returns repositories with empty names, if the project's namespace is used as the repository`